src/bundle/Resources/public/js/scripts/admin.context.menu.js

```diff
@@ -5,8 +5,12 @@
         return;
     }
 
-    const menuButtons = [...adapatItemsContainer.querySelectorAll('.ibexa-context-menu__item > .ibexa-btn:not(.ibexa-btn--more)')];
-    const popupMenuElement = adapatItemsContainer.querySelector('.ibexa-popup-menu');
+    const menuButtons = [
+        ...adapatItemsContainer.querySelectorAll(
```

Comment:
I see that's not your variable, but there misspell in name, so maybe you could fix it? :)

Reply:
I cannot even figure it out what should be real name.
Adapt, adapted?

Reply:
It suppose to be adapted I think but not sure 😕